libnavigation-core/src/test/java/com/mapbox/navigation/core/navigator/NavigatorMapperTest.kt

```diff
@@ -0,0 +1,179 @@
+package com.mapbox.navigation.core.navigator
+
+import android.location.Location
+import com.mapbox.navigation.base.trip.model.RouteProgress
+import com.mapbox.navigation.core.trip.session.MapMatcherResult
+import com.mapbox.navigation.navigator.internal.TripStatus
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert
+import org.junit.Test
+
+class NavigatorMapperTest {
+
+    private val enhancedLocation: Location = mockk(relaxed = true)
+    private val keyPoints: List<Location> = mockk(relaxed = true)
+    private val routeProgress: RouteProgress = mockk(relaxed = true)
+    private val offRoute: Boolean = mockk(relaxed = true)
+
+    @Test
+    fun `map matcher result sanity`() {
+        val tripStatus = TripStatus(
+            enhancedLocation,
+            keyPoints,
+            routeProgress,
+            offRoute,
+            mockk {
+                every { offRoadProba } returns 0f
+                every { map_matcher_output } returns mockk {
+                    every { isTeleport } returns false
+                    every { matches } returns listOf(
+                        mockk {
+                            every { proba } returns 1f
+                        }
+                    )
+                }
+            }
+        )
+        val expected = MapMatcherResult(
+            enhancedLocation,
+            keyPoints,
+            isOffRoad = false,
+            offRoadProbability = 0f,
+            isTeleport = false,
+            roadEdgeMatchProbability = 1f
+        )
+
+        val result = tripStatus.getMapMatcherResult()
+
+        Assert.assertEquals(expected, result)
```

Comment:
NIT

```suggestion
        assertEquals(expected, result)
```

Same thing applies to the rest of the tests 👇